enterprise/models.py

```diff
@@ -2876,3 +2877,109 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+@python_2_unicode_compatible
+class AdminNotificationFilter(TimeStampedModel):
+    """
+    Model for Admin Notification Filters.
+
+    .. no_pii:
+    """
+    filter = models.CharField(
+        max_length=50,
+        blank=False, null=False, unique=True,
+        help_text=_("Filters to show banner notifications conditionally.")
+    )
+
+    class Meta:
+        app_label = 'enterprise'
+        verbose_name = _('Admin Notification Filter')
+        verbose_name_plural = _('Admin Notification Filters')
+        ordering = ['filter']
+
+    def __str__(self):
+        """
+        Return human-readable string representation.
+        """
+        return self.filter
+
+    def __repr__(self):
+        """
+        Return uniquely identifying string representation.
+        """
+        return self.__str__()
+
+
+@python_2_unicode_compatible
+class AdminNotification(TimeStampedModel):
+    """
+    Model for Admin Notification.
+
+    .. no_pii:
+    """
+    text = models.CharField(
+        max_length=255,
+        blank=False, null=False,
+        help_text=_('Notification banner which will appear '
+                    'to enterprise admin on admin portal.')
+    )
+
+    admin_notification_filter = models.ManyToManyField(AdminNotificationFilter, blank=True,
+                                                       related_name='notification_filter')
+
+    is_active = models.BooleanField(default=True)
+    start_date = models.DateField(default=timezone.now)
+    expiration_date = models.DateField(default=timezone.now)
+
+    class Meta:
+        app_label = 'enterprise'
+        verbose_name = _('Enterprise Customer Admin Notification')
+        verbose_name_plural = _('Enterprise Customer Admin Notifications')
+        ordering = ['start_date']
+
+    def __str__(self):
+        """
+        Return human-readable string representation.
+        """
+        return self.text
+
+    def __repr__(self):
+        """
+        Return uniquely identifying string representation.
+        """
+        return self.__str__()
+
+
+@python_2_unicode_compatible
+class AdminNotificationRead(TimeStampedModel):
+    """
+    Model for Admin Notification Read Status.
+
+    .. no_pii:
+    """
+    enterprise_customer_user = models.ForeignKey(
```

Comment:
Has this been confirmed that the staff/admin users are added in this same model? Confused because I can't see any field in this model that differentiates staff users from learners.

Reply:
Yes, Admin users will be added in the same mode. Its the `enterprisefeatureuserroleassignment` model where user type is stored.